docs/Clearwater_Stateful_Statistics.md

```diff
@@ -9,7 +9,7 @@ These statistics are accessible in the same manner as our standard SNMP statisti
 
 ## Usage
 
-As these statistics are representative of deployment wide state, their usage is slightly different to the standard per-node SNMP statistics. They key difference is that, due to the redundancy design in Chronos, the statistics are subject to a replication factor (which is by default set to 2). To accurately report the statistics from a whole deployment, the individual values should be gathered from each node in the deployment, and then the sum total should be divided by the replication factor.
+As these statistics are representative of deployment wide state, their usage is slightly different to the standard per-node SNMP statistics. They key difference is that, due to the redundancy design in Chronos, the statistics are subject to a replication factor (which is by default set to 2). To accurately report the statistics from a whole single site deployment, the individual values should be gathered from each node in the deployment, and then the sum total should be divided by the replication factor.
```

Comment:
Can you remove the word whole? I think that makes this read better.

Reply:
done